utils/gyb.py

```diff
@@ -655,7 +656,9 @@ def execute(self, context):
         # Execute the code with our __children__ in scope 
         context.localBindings['__children__'] = self.children
         result = eval(self.code, context.localBindings)
-        assert context.localBindings['__children__'] is self.children
+
+        if context.localBindings['__children__'] is not self.children:
+            raise ValueError("The code is not allowed to mutate children")
```

Comment:
This should say `__children__`, with underscores, as it refers to the actual variable name.

Could you add tests that I mentioned in the other messages?  gyb has a `--test` mode.